package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "adios-v1",
-  "version": "0.0.0",
+  "name": "adios-v1.1",
```

Comment:
We should probably rename this to just "adios", since the version name shouldn't be in the package name.

Reply:
```suggestion
  "name": "adios",
```